indexer/services/ender/src/scripts/handlers/dydx_funding_handler.sql

```diff
@@ -81,7 +82,7 @@ BEGIN
                 funding_update_response = jsonb_set(funding_update_response, ARRAY[(funding_index_updates_record."perpetualId")::text], dydx_to_jsonb(funding_index_updates_record));
 
             ELSE
-                errors_response = array_append(errors_response, 'Received unknown FundingEvent type.');
+                errors_response = array_append(errors_response, '"Received unknown FundingEvent type."'::jsonb);
```

Comment:
without adding this jsonb, is this error message not showcasing properly?

Reply:
this code path was never tested, and as is, throws an error.